cooldowns_paladin.lua

```diff
@@ -143,9 +140,6 @@ LCT_SpellData[4987] = {
 LCT_SpellData[6940] = {
 	class = "PALADIN",
 	defensive = true,
-	specID = { SPEC_PALADIN_HOLY },
-	opt_charges = 2,
-	opt_charges_linked = { 1044, 1022, 204018 },

```

Comment:
please remove `6940` from the `opt_charges_linked` in Hand of Freedom

Reply:
Done.

I have a question about code like the following (from Hand of Freedom section):
cooldown_overload = {
		[SPEC_PALADIN_RETRIBUTION] = 20, -- ret
The only code where cooldown_overload is referenced is in function GetCooldownTime, it seems to me that the overloads are unconditionally applied if it's the spec.
I might be missing something here. Could you explain how cooldown_overload works?

Reply:
> it seems to me that the overloads are unconditionally applied if it's the spec.

They are, `cooldown_overload` is not meant for (pvp) talents, only for cd reductions you get from passives in your spec.

Reply:
I see. Thanks for the explanation :)

I think at some point I'll go through the cd reduction passives for specs and update those, at least the ones I consider high impact.